microservices/api/src/controllers/ConfigurationController.php

```diff
@@ -21,9 +20,18 @@ public function actionCountries(
 
     public function actionLocales(
         ConfigurationProvider $configurationProvider,
-        Response $response
-    ): array|string {
+        Response              $response
```

Comment:
Don't do manual indenting like this, we use a tool for this : vendor/bin/ecs -- easy coding standard.